app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,20 @@
+class ReviewsController < ApplicationController
+  before_action :set_user
+
+  def create
+    @review = Review.new(review_params)
+    @review.creator = current_user
+    @review.destinator = @user
+    @review.save ? (redirect_to root) : (render user_path(@user))
```

Comment:
why redirect to root?
maybe redirect to: user who receives the review profile page?